007_smoothed_Laplace/smoothed_laplace.ipynb

```diff
@@ -162,14 +167,14 @@
     "    laplace_grad_values[i] = (laplace.logpdf(x[i])[0]-laplace.logpdf(x[i]-dx)[0])/dx\n",
     "\n",
     "plt.figure()\n",
-    "plt.subplot(1,2,1)\n",
+    "plt.subplot(2,1,1)\n",
     "plt.plot(x, laplace_pdf_values, '-o', label='Laplace')\n",
     "plt.plot(x, smoothed_laplace_pdf_values, label='Smoothed Laplace')\n",
     "plt.xlabel('x')\n",
     "plt.ylabel('value')\n",
     "plt.title('Log PDF')\n",
     "plt.legend(loc='upper right')\n",
-    "plt.subplot(1,2,2)\n",
+    "plt.subplot(2,1,2)\n",
```

Comment:
Better, but still squashed when using subplot.  Can you change to avoid subplots and be in two separete plt.figure().

Reply:
Done